File: eval/sample_project_data/mixed.py
import os
from pathlib import Path

def load_config(path):
    with open(path) as f:
        return f.read()

def save_result(data, filename):
    Path(filename).write_text(data)

class DataProcessor:
    def process(self, raw):
        return raw.strip().lower()

    def validate(self, data):
        return len(data) > 0

File: eval/sample_java_project_data/Mixed.java
import java.nio.file.Files;
import java.nio.file.Paths;

public class Mixed {
    public static String loadConfig(String path) throws Exception {
        return Files.readString(Paths.get(path));
    }

    public static void saveResult(String data, String filename) throws Exception {
        Files.writeString(Paths.get(filename), data);
    }

    public static class DataProcessor {
        public String process(String raw) {
            return raw.trim().toLowerCase();
        }

        public boolean validate(String data) {
            return data.length() > 0;
        }
    }
}
